CMakeLists.txt

```diff
@@ -15,6 +15,13 @@ project(
 option(VIAM_UR_USE_WALL_WERROR "Build with -Wall and -Werror flags" ON)
 
 
+# - `VIAM_UR_DISABLE_APPIMAGE`
+#
+# Configure the build for not producing an AppImage
+#
+option(VIAM_UR_DISABLE_APPIMAGE "Disable AppImage builds" OFF)
```

Comment:
Feature flag for this while we iterate on packaging, it defaults OFF so appimage builds are still the default.